docs/how-to/import-prisma-schema.md

```diff
@@ -0,0 +1,120 @@
+
+This way you don't have to manually create entities, fields, and relationships from scratch saving you a lot of time.
+
+Just upload your `schema.prisma` file.
+Amplication will then search for models and [convert them into Amplication's internal structure](/getting-started/how-prisma-schema-is-converted-into-entities).
+
+This guide walks you through how to generate a custom `schema.prisma` file from your existing database and upload it to Amplication.
+
+## How to create and upload your `schema.prisma` file
+
+First, you will run the Prisma introspection process on your existing database.
+The introspection process will generate a schema file representing your database's tables and columns.
+
+Next, you'll upload the generated file into Amplication, ensuring it's free of errors before doing so.
+After upload, you'll see an upload log of everything that happens to [convert the schema into Amplication's internal structure](/getting-started/how-prisma-schema-is-converted-into-entities).
+If any warnings appear in the upload log you can then choose to modify your `schema.prisma` file and re-upload it or continue as is.
+
+### Step 1: Run the Introspection process
+
+Introspection is a feature of Prisma that scans your entire database and generates a `schema.prisma` file.
+This feature generates a data model that reflects your current database schema.
+
+Here's a quick overview of its main functions on SQL databases:
+
+- Map tables in the database to Prisma models 
+- Map columns in the database to the fields of Prisma models
+- Map indexes in the database to indexes in the Prisma schema  
+- Map database constraints to attributes or type modifiers in the Prisma schema
+
+To learn more see [Prisma's Introspection docs](https://www.prisma.io/docs/concepts/components/introspection#what-does-introspection-do).
+
+Run the `prisma db pull` command in the terminal to start the introspection process.
+Using this command requires your database's **Connection URL** be set in your Prisma Schema `datasource`.
+
+```bash
+npx prisma db pull
+```
+
+:::caution
+After generating your schema file, double check it in your code editor for errors.
+Make sure you fix any errors before you upload your `schema.prisma` file into Amplication.
+There are some cases where the introspection process ends with a schema that might have errors. 
+:::
+
+Here's an example of where the generated schema file could have an error:
+
+```graphql title="schema.prisma"
+model searchText {
+  application_id   BigInt  @id @default(0) @db.UnsignedBigInt
+  guid             String  @db.VarChar(50)  
+  application_text String? @db.LongText   
+
+  @@index([application_text], map: "idx_searchText_application_text")
+}
+```
+
+In this case, it's because you cannot define an index on fields with a native type of **LongText** on MySQL. You have to use the `length` argument to the field in the index definition to allow this.
```

Comment:
I dont think we should give an example of error, our example should be of a successful use case